administrator/components/com_installer/src/Model/UpdateModel.php

```diff
@@ -426,12 +430,15 @@ private function install($update)
                 $url .= $extra_query;
             }
 
+            $errorUrlLinks[] = sprintf($urlLink, $url);
+
             $mirror++;
         }
 
         // Was the package downloaded?
         if (!$p_file) {
-            Factory::getApplication()->enqueueMessage(Text::sprintf('COM_INSTALLER_PACKAGE_DOWNLOAD_FAILED', $url), 'error');
+            Factory::getApplication()->enqueueMessage(Text::sprintf('COM_INSTALLER_PACKAGE_DOWNLOAD_FAILED', $url),
+                implode('<br>', $errorUrlLinks)), 'error');
```

Comment:
```suggestion
                implode('<br>', $errorUrlLinks), 'error');
```
This is a PHP syntax error! Authors of pill requests are supposed to test their pull requests themselves before submitting them. Obviously this was not the case here.

Reply:
Not knowing enough about GitHub, I assume the code changes that you posted above @richard67 aren't live yet, there must be a step by the original PR maker BrainforgeUK for it be incorporated in the Patch?

I ask because I just re-tested and it still crashes with the same exact error.

Reply:
@exlemor Sure. He has to accept my suggestion so it will be shown here as resolved.

Reply:
OK, thanks for that precision.